src/parse.ts

```diff
@@ -0,0 +1,40 @@
+import type { VALIDATORS, ValidatorToType } from './validators';
+import { NUMBER_VALIDATOR, DATE_VALIDATOR, STRING_VALIDATOR } from './validators';
+
+const simplifiedISODateStringRegex = /^[+-]?\d{4}/;
+export const isISODateString = (value: unknown) =>
+  simplifiedISODateStringRegex.test(value as string)
+    ? !Number.isNaN(Number(new Date('1970-01-01T00:00:00.000Z')))
```

Comment:
Let's add a test for this
```suggestion
    ? !Number.isNaN(Number(new Date(value)))
```